crates/qpy/src/consts.rs

```diff
@@ -0,0 +1,70 @@
+// This code is part of Qiskit.
+//
+// (C) Copyright IBM 2025
+//
+// This code is licensed under the Apache License, Version 2.0. You may
+// obtain a copy of this license in the LICENSE.txt file in the root directory
+// of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+//
+// Any modifications or derivative works of this code must retain this
+// copyright notice, and modified files need to carry a notice indicating
+// that they have been altered from the originals.
+
+use qiskit_circuit::operations::StandardGate;
+
+pub fn standard_gate_from_gate_class_name(name: &str) -> Option<StandardGate> {
```

Comment:
It would be good to add a Rust test here to catch any possible (even if unlikely) change in the list of our supported `StandardGate`s.